src/ui/osx/TogglDesktop/AboutWindowController.m

```diff
@@ -55,6 +63,12 @@ - (void)windowDidLoad
 
 - (BOOL)updateCheckEnabled
 {
+	if (self.unsupportedOS)
```

Comment:
Then, reusing the logic in `UnsupportedNotice`
It could be
```objective-c
if ([[UnsupportedNotice shared] unsupportedOS])  {
    // Your logic
}
```